fineract-provider/src/main/java/org/apache/fineract/portfolio/client/api/ClientsApiResourceSwagger.java

```diff
@@ -252,6 +252,8 @@ private GetClientsGroups() {}
         public List<GetClientsGroups> groups;
         @Schema(example = "test@test.com")
         public String emailAddress;
+        @Schema(example = "123")
+        public String externalId;
```

Comment:
I think more than 2 API should retrieve the external id as well. It should be retrieved in all cases (where it makes sense). Like: GetClientsResponse

Reply:
Done